packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -82,18 +111,13 @@ enum PollState {
  * @type TokenRatesState
  *
  * Token rates controller state
- * @property contractExchangeRates - Hash of token contract addresses to exchange rates (single globally selected chain, will be deprecated soon)
- * @property contractExchangeRatesByChainId - Hash of token contract addresses to exchange rates keyed by chain ID and native currency (ticker)
+ * @property marketData - Hash of token contract addresses to market data
```

Comment:
Hmm, this is a bit more than that. It seems like this collection is still keyed by chain ID, but then after that, it is keyed by token contract address. What about:

``` suggestion
 * @property marketData - The market data for tokens on a chain.
```

or, if you want to be more specific:

``` suggestion
 * @property marketData - Market data for tokens, keyed by chain ID and then token contract address.
```